### ISSUE TITLE
Return immutable tuple instead of list

### ISSUE THREAD

**Issue by CremaLuca** [Slthy/Microplus-Lenex#13]
Nella funzione `utils.url` restituisci una lista mutabile
https://github.com/Slthy/Microplus-Lenex/blob/b6380314e274b9a568c6320ebd00d117442f74a9/utils.py#L85

Siccome sarà sempre lunga 2 consiglio ritornare una tupla (immutabile) così anche da facilitarne l'unpacking:
https://github.com/Slthy/Microplus-Lenex/blob/b6380314e274b9a568c6320ebd00d117442f74a9/functions.py#L12-L14
se restituisci una tupla può diventare
```py
base_url, event = utils.url(url)
```

Basta che cambi le parentesi da quadre a tonde

**Comment by Slthy** [comment]
vero, ma a questo punto preferisco modificare direttamente la stringa, piuttosto che usare una funzione.
